app/src/main/java/com/example/caloriecounter/feature/restaurant/ui/categories/CategoriesContract.kt

```diff
@@ -5,21 +5,20 @@ import com.example.caloriecounter.base.BaseSideEffect
 import com.example.caloriecounter.base.BaseState
 import com.example.domain.models.Category
 
-class CategoriesContract {
-    sealed class Event : BaseEvent {
-        data class SelectedCategory(val category: String) : Event()
-        object ReloadData : Event()
-    }
+sealed class CategoriesEvent : BaseEvent {
+  data class SelectedCategory(val category: String) : CategoriesEvent()
+  object ReloadData : CategoriesEvent()
+}
```

Comment:
Good, but you can also divide Event, State and Effect to different files

Reply:
I think for now it's not necessary. If the following events have more parameters, I will separate them.